tools/gometalinter.json

```diff
@@ -6,5 +6,5 @@
     "Deadline": "500s",
     "Vendor": true,
     "Cyclo": 11,
-    "Exclude": ["/usr/lib/go/src/", "client/lcd/statik/statik.go"]
+    "Exclude": ["/usr/lib/go/src/", "client/lcd/statik/statik.go", "vendor/*"]
```

Comment:
Is this redundant with `"Vendor": true` ?

Reply:
Nope, see above